liquibase-standard/src/main/java/liquibase/changelog/AbstractChangeLogHistoryService.java

```diff
@@ -124,6 +125,14 @@ public void replaceChecksum(ChangeSet changeSet) throws DatabaseException {
         reset();
     }
 
+    @Override
+    public void replaceFilePath(ChangeSet changeSet, String oldPath) throws DatabaseException {
```

Comment:
Is there any way to make this more targeted? And can we add a log about what's happening?

Reply:
I added another check in the calling class. Also, logs added!